### ISSUE TITLE
Web support?

### ISSUE THREAD

**Issue by Bilonik** [aloisdeniel/flutter_geocoder#39]

**Comment by stevenosse** [comment]
Since this package uses platform specific code and features i don't think this will be available soon...

**Comment by aloisdeniel** [comment]
Yep, the only web browser API available seems to be geolocation (https://developer.mozilla.org/fr/docs/Web/API/Geolocation_API).

**Comment by giorgio79** [comment]
You can use Google Geocoding API. A simple http call...
https://developers.google.com/maps/documentation/geocoding/overview

**Comment by tehsunnliu** [comment]
Hi, In `geocoder` package if we use `Geocoder.google(...)` it uses Dart's `HttpClient` to forward and reverse geocoding. However, the dart `HttpClient` doesn't support the web. I've created a [PR](https://github.com/aloisdeniel/flutter_geocoder/pull/72) fixing this issue adding support for the web.
